### PR TITLE
Publish tests junit

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -70,8 +66,11 @@ pipeline {
                       }
                       sh '''curl -X POST -H 'Content-type: application/json' --data-binary "@data.txt" https://hooks.slack.com/services/${SLACK_TOKEN}'''
                 }
+
```
Comment (nmagnezi) [inline Jenkinsfile:69]: ¯\\_(ツ)_/¯

**Review by nmagnezi — CHANGES_REQUESTED** [review]
I don't see tests reported in [ci/prow/unit-test](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-installer-agent/202/pull-ci-openshift-assisted-installer-agent-master-unit-test/1400361363158601728)

**Comment by YuviGold** [comment]
> I don't see tests reported in [ci/prow/unit-test](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-installer-agent/202/pull-ci-openshift-assisted-installer-agent-master-unit-test/1400361363158601728)

You're right. https://github.com/openshift/release/pull/18987 would fix that.

**Review by osherdp — APPROVED** [review]

**Pull request merged**